README.md

```diff
@@ -1,44 +1,140 @@
+
+You can find the endpoints from our microservice listed in the README for that project [here](https://github.com/jpc20/obituary_microservice)
+
+### NYT Endpoints:
+
+All requests below are tacked onto the end of this root URL: [https://api.nytimes.com/svc/search/v2//articlesearch.json](https://api.nytimes.com/svc/search/v2//articlesearch.json)
 
-## Microservice
+  Returns all obituaries:
 
-## Optimization
+    `GET ?fq=new_desk:(obituaries)&fq=section_name:(obituaries)`
+
+  Returns obituaries that contain the name provided:
+
+    `GET ?fq=headline:("#{name}") AND section_name:(obituaries)&q=relevance`
+
+  Returns obituaries for individuals who died from COVID-19:
+
+    `GET ?fq=section_name:(obituaries)&q=coronavirus`
+
+  Returns obituaries that were published on a particular date:
+
+    `GET ?fq=section_name:(obituaries)&facet_fields=pub_year&facet=true&begin_date="#{beginning_date}"&ending_date="#{ending_date}"`
+
+  Returns most recently published obituaries:
+
+    `GET ?fq=section_name:(obituaries)&sort=newest`
+
+  Returns obituaries that match the name and year provided:
+
+    `GET ?fq=section_name:(obituaries) AND headline:("#{name}") AND pub_year:("#{year}")`
+
+  Returns obituaries that were published during a given year:
+
+    `GET ?fq=section_name:(obituaries) AND pub_year:("#{year}")`
+
+## Optimization <a name="optimization"></a>
+
+The requirements of this project dictated we needed to implement one of three different types of optimization. We chose to focus on caching. All obituaries from our database are cached so that when a page is reloaded of the obituary appears on another page the results have already been cached and the website loads quickly because it is not making a redundant request of our database. The caching portion of our project was implemented single-handedly by [Eric Larson](https://github.com/EricLarson2020).
 
 ## Extensions
 
-## Implementation Instructions
+The extensions that we were able to complete include:
+  * Automated emails
+  * Google Oauth
+
+Future iterations:
+  * Admin is able to designate an obituary "Editor's Choice"
+  * Tagging and search by category
+  * Import facebook posts
+  * Link together multiple obituaries and tributes for the same deceased person
+  * Events for funerals and memorials
+  * Visitors can leave comments
+  * Obituaries by location display on a map
+  * Subscribe to notifications for a particular zip code
+
+## Testing
+
+For testing we used RSpec with [Capybara](https://github.com/teamcapybara/capybara), [Shoulda-Matchers](https://github.com/thoughtbot/shoulda-matchers), [FactoryBot](https://github.com/thoughtbot/factory_bot_rails) and [Faker](https://github.com/faker-ruby/faker) gems, as well as [SimpleCov](https://github.com/colszowka/simplecov) to determine our test coverage and pinpoint areas that were overlooked.
+
+As of writing this, our test coverage is INSERT PERCENT.
+Badges for testing from TravisCI???
```

Comment:
These we will need to add once the project is finished!

Reply:
Woot woot I think the badge idea is really cool, good suggestion from Dione.